### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by manitghogar**
@arjunblj Please review code ex1 and ex2

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ewhfahwefheoifhw
```
Comment (arjunblj) [inline test.txt:1]: You can delete this file as this is just a test file at the top of your repository that doesn't need to be committed.

You can actually add commits to your PR even after you open it, it's functionally a "work in progress" until it's closed (without any merge) or merged into `master` (or whatever branch you want to merge into). This means that you can just commit a change to this and delete this file.

File: 2/ex2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# A comment, this is so you can read your program later.
+# Anything after the # is ignored by python.
+
+print "I could have code like this." # and the comment after is ignored
```
Comment (arjunblj) [inline 2/ex2.py:4]: Awesome, glad you got to see that comments/code can live in conjunction like this. Two (quick) exercises for you (no need to commit anything):

1.Try writing and running something like this in a separate file:

``` python
print '5'
"""Example comment."""
```

Is this valid? What does that second line do? It seems just like a comment! What is this useful for? The name for this is a `docstring` and you can actually write comments on multiple lines, without prefacing all of them with `#`. It's very useful for writing multi-line comments.
<br />2. What do you think comments are useful for? The obvious reason is to explain bits of your code that are confusing but do any other reasons come to mind?

File: 1/ex1.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+print "Hello World!"
+#print "Hello Again"
```
Comment (arjunblj) [inline 1/ex1.py:2]: Try highlighting all of these lines in Sublime and hit `Cmd + /`. Try it again, what did it do?

**Comment by arjunblj** [comment]
One last thing (and this is a bit more divisive) to note is that different developers have different thoughts around git hygiene (that is, "what should my git commits look like?"). It isn't ultra-important by any means, particularly when you're working with just yourself on a super small toy project but it's a good habit to get into.

This is a really good post (really, it's the blog post I point _every single person_ to). The first part isn't important (yet) but Cmd+F for "The seven rules of a great git commit message". Those 7 rules are super helpful and I follow them the majority of the time, I recommend you do too. Scrolling past that explains what each of the seven rules is if you're unsure. :100:

**Comment by arjunblj** [comment]
Oops, realized I forgot to link to the post I had mentioned above: http://chris.beams.io/posts/git-commit/

File: 1/ex1.py (hunk 3 above)
Comment (manitghogar) [inline 1/ex1.py:2]: cmd + / brought up the help menu?

File: 2/ex2.py (hunk 2 above)
Comment (manitghogar) [inline 2/ex2.py:4]: Tried the docstring example - I see what you mean in it removing the need for having to preface every line with a #.

Comments I believe are also good for ignoring certain lines of code when running the program to see how that particular line of code affects the entire program.

**Pull request merged**